src/relay/azext_relay/_params.py

```diff
@@ -0,0 +1,85 @@
+
+    # region WCF Relay Authorizationrule
+    for scope in ['relay wcfrelay authorizationrule', 'relay wcfrelay authorizationrule keys list', 'relay wcfrelay authorizationrule keys renew']:
+        with self.argument_context(scope) as c:
+            c.argument('authorization_rule_name', options_list=['--name', '-n'], help='name of the WCF Relay AuthorizationRule')
+            c.argument('relay_name', options_list=['--wcfrelay-name'], help='name of the WCF Relay')
+
+    with self.argument_context('relay wcfrelay authorizationrule create') as c:
+        c.argument('rights', options_list=['--access-rights'], arg_type=get_enum_type(['Send', 'Listen', 'Manage']), help='AuthorizationRule rights of type list')
+
+    with self.argument_context('relay wcfrelay authorizationrule keys renew') as c:
+        c.argument('key_type', options_list=['--key-name'], arg_type=get_enum_type(['PrimaryKey', 'SecondaryKey']))
+
+
+# region - Hybrid Connection Create
+def load_arguments_hybridconnections(self, _):
+    with self.argument_context('relay hybrid-connections') as c:
+        c.argument('hybrid_connection_name', options_list=['--name', '-n'], help='Name of Hybrid Connection')
+
+    with self.argument_context('relay hybrid-connections create') as c:
+        c.argument('requires_client_authorization', options_list=['--requires-client-authorization'], type=bool, help='True if client authorization is needed for this relay; otherwise, false.')
+        c.argument('user_metadata', options_list=['--user-metadata'], help='The usermetadata is a placeholder to store user-defined string data for the Hybrid Connection endpoint. For example, it can be used to store descriptive data, such as list of teams and their contact information. Also, user-defined configuration settings can be stored.')
+
+    # region Hybrid Connection Authorizationrule
+    for scope in ['relay hybrid-connections authorizationrule', 'relay hybrid-connections authorizationrule keys list', 'relay Hybrid Connection authorizationrule keys renew']:
+        with self.argument_context(scope) as c:
+            c.argument('authorization_rule_name', options_list=['--name', '-n'], help='name of the Hybrid Connection AuthorizationRule')
+            c.argument('hybrid_connection_name', options_list=['--hybrid-connection-name'], help='name of the Hybrid Connection')
+
+    with self.argument_context('relay hybrid-connections authorizationrule create') as c:
+        c.argument('rights', options_list=['--access-rights'], arg_type=get_enum_type(['Send', 'Listen', 'Manage']), help='AuthorizationRule rights of type list')
+
+    with self.argument_context('relay hybrid-connections authorizationrule keys renew') as c:
+        c.argument('key_type', options_list=['--key-name'], arg_type=get_enum_type(['PrimaryKey', 'SecondaryKey']))
```

Comment:
key_type should also be made into an arg_type and the definition reused.

Reply:
implemented